scss/detail.scss

```diff
@@ -5,12 +5,22 @@
   font-size: 18px;
 }
 
+@mixin box-shadow {
```

Comment:
This mixin is only being used once, i dont see the need for it. Sometimers the shadows are repeated throughout the app, if that is the case here you should make a more general file with this mixin

Reply:
I'm going to solve this in my next PR since I'm gonna be re-using it there.